### PR TITLE
[DOCFIX] Fix broken links

### PR DESCRIPTION
**Pull request by ns1123**
Fixing broken links in 1.8 docs

### CODE REVIEW

**Comment by ns1123** [comment]
@ZacBlanco PTAL

**Comment by ZacBlanco** [comment]
jenkins, test this please

File: docs/_data/table/en/worker-configuration.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ alluxio.worker.data.port:
...
   'Selects the networking stack to run the worker with. Valid options are: `alluxio.worker.netty.NettyDataServer`.'
 alluxio.worker.data.server.domain.socket.address:
-  'The path to the domain socket. Short-circuit reads make use of a UNIX domain socket when this is set (non-empty). This is a special path in the file system that allows the client and the AlluxioWorker to communicate. You will need to set a path to this socket. The AlluxioWorker needs to be able to create the path. If alluxio.worker.data.server.domain.socket.as.uuid is set, the path should be the home directory for the domain socket. The full path for the domain socket with be <path>/<uuid>.'
+  'The path to the domain socket. Short-circuit reads make use of a UNIX domain socket when this is set (non-empty). This is a special path in the file system that allows the client and the AlluxioWorker to communicate. You will need to set a path to this socket. The AlluxioWorker needs to be able to create the path. If alluxio.worker.data.server.domain.socket.as.uuid is set, the path should be the home directory for the domain socket. The full path for the domain socket with be {path}/{uuid}.'
```
Comment (ZacBlanco) [inline docs/_data/table/en/worker-configuration.yml]: This is an autogenerated file. Please modify the description of the property key in `PropertyKey.java` and then run `./bin/alluxio confDocGen`

File: docs/en/operation/Journal.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ backup directory can be configured by setting `alluxio.master.backup.directory`
 alluxio.master.backup.directory=/alluxio/backups
 ```
 
-See the [backup command documentation]({{ '/en/operation/Admin-CLI.html' | relativize_url }}#backup)
+See the [backup command documentation]({{ '/en/operation/Admin-CLI.html#backup' | relativize_url }}#backup)
```
Comment (apc999) [inline docs/en/operation/Journal.md]: why two `#backup`?

Comment (ns1123) [reply docs/en/operation/Journal.md]: my fault. I just copied the enterprise PR without looking. Fixed

File: docs/_data/table/en/worker-configuration.yml (hunk 1 above)
Comment (ns1123) [reply docs/_data/table/en/worker-configuration.yml]: Fixed.

**Comment by LuQQiu** [comment]
jenkins, test this please

**Comment by ZacBlanco** [comment]
jenkins, test this please

**Comment by LuQQiu** [comment]
I am fixing the flaky test in https://github.com/Alluxio/alluxio/pull/9184. Feel free to merge this PR if block on test `AbstractFileSystemTest.cannotInitializeWithoutMasterHostname Expected exception: java.lang.IllegalArgumentException`

**Comment by ns1123** [comment]
jenkins, test this please

**Comment by ns1123** [comment]
@apc999 can you re-add the 'Doc fix' label and merge this please?

**Comment by apc999** [comment]
@ns1123 do we need to apply the same patch to master branch?

**Comment by LuQQiu** [comment]
@apc999 The flaky test only exists in branch 1.8.

**Comment by ns1123** [comment]
@apc999 I'm going to remove the commit with the test fix. Lu will merge that in a separate [PR](https://github.com/Alluxio/alluxio/pull/9184)

**Comment by apc999** [comment]
@ns1123  can you resolve the conflict?

**Comment by ns1123** [comment]
@apc999 done

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Pull request merged**